osu.Game.Rulesets.Mania/Skinning/LegacyStageBackground.cs

```diff
@@ -58,5 +60,7 @@ protected override void Update()
             if (rightSprite?.Height > 0)
                 rightSprite.Scale = new Vector2(1, DrawHeight / rightSprite.Height);
         }
+
+        public void AddColumnBackground(Drawable background) => columnBackgroundContainer.Add(background);
```

Comment:
Was thinking this should just be a `.Children =` but i guess this is local enough to not really matter.

Reply:
It can't be. Columns are added in sequence, not all at once.

Reply:
right, i see. it's shared between all columns here. bit of a convoluted situation but visually looks correct and is legacy class.